vochain/indexer/indexer.go

```diff
@@ -329,26 +330,6 @@ func (idx *Indexer) AfterSyncBootstrap() {
 func (idx *Indexer) Commit(height uint32) error {
 	idx.lockPool.RLock()
 	defer idx.lockPool.RUnlock()
-	// Add Entity and register new active process
-	for _, p := range idx.newProcessPool {
-		if err := idx.newEmptyProcess(p.ProcessID); err != nil {
-			log.Errorw(err, "commit: cannot create new empty process")
-			continue
-		}
-		if !idx.App.IsSynchronizing() {
-			idx.addProcessToLiveResults(p.ProcessID)
-		}
-		log.Debugw("new process", "processID", hex.EncodeToString(p.ProcessID))
-	}
-
-	// Update existing processes
-	for _, p := range idx.updateProcessPool {
-		if err := idx.updateProcess(p); err != nil {
-			log.Errorw(err, "commit: cannot update process")
-			continue
-		}
-		log.Debugw("updated process", "processID", hex.EncodeToString(p))
-	}
```

Comment:
the sql transaction at blockTx should be committed here, in the Commit method, unless I'm missing something about your approach. if you commit every time you call newEmptyProcess, then we're writing to the DB before the entire block is committed here.